integration-tests/pom.xml

```diff
@@ -91,6 +91,11 @@
             <artifactId>maven-surefire-plugin</artifactId>
             <configuration>
               <skipTests>${native.surefire.skip}</skipTests>
+              <environmentVariables>
+                <NAMESPACE_FROM_ENV>fromEnvVarNS</NAMESPACE_FROM_ENV>
+                <QUARKUS_OPERATOR_SDK_CONTROLLERS_EMPTY_NAMESPACES>fromEnv1,  fromEnv2</QUARKUS_OPERATOR_SDK_CONTROLLERS_EMPTY_NAMESPACES>
+                <VARIABLE_NS_ENV>variableNSFromEnv</VARIABLE_NS_ENV>
```

Comment:
Let's try that first and then you can also try to add them to the `systemPropertyVariables` block below.